extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -199,9 +199,7 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     if (!this.iframeLayoutBox_) {
       this.measureIframeLayoutBox_();
```

Comment:
`measureIframeLayoutBox_()` returns iframe layoutRect relative to the `amp-ad` or `amp-iframe` element. However I searched our code and it seems to me the `this.iframeLayoutBox_` is only used here. So instead of applying box offset and apply it back here. Why don't we just return layoutRect relative to doc in `#measureIframeLayoutBox`